Networking/NetworkingTests/Responses/systemStatus-without-data.json

```diff
@@ -0,0 +1,485 @@
+{
+    "data": {
```

Comment:
❓ is this JSON response supposed to have the `data` envelope?

Reply:
Thanks for catching! Fixed in a9b489ba2e7791329e8bec3bb95be2528b7186a7.